ledger/error/src/main/scala/com/daml/error/ErrorCategory.scala

```diff
@@ -360,6 +361,23 @@ object ErrorCategory {
       )
       with ErrorCategory
 
+  @Description(
+    """The mutable state of the system no longer satisfies the requirements set by the submission."""
+  )
+  @RetryStrategy("""Retry quickly (indefinitely or limited).""")
+  @Resolution("""Expectation: this is processing-flow level contention that should be handled by
+                |retrying the request with appropriate backoff.""")
+  object InconsistentSystemStateDuringSubmission
+      extends ErrorCategoryImpl(
+        grpcCode = Some(Code.ABORTED),
+        logLevel = Level.INFO,
+        retryable = None,
+        securitySensitive = false,
+        asInt = 15,
+        rank = 3,
+      )
+      with ErrorCategory
```

Comment:
The only concern I have with this is that so far we have one category per gRPC status code we decided to use. I think we should reuse `ContentionOnSharedResources`.

Reply:
Done